open-metadata-implementation/governance-servers/open-lineage-services/open-lineage-services-api/src/main/java/org/odpi/openmetadata/governanceservers/openlineage/model/Queries.java

```diff
@@ -3,6 +3,8 @@
 package org.odpi.openmetadata.governanceservers.openlineage.model;
 
 public enum Queries {
```

Comment:
Convention is for singular for class name and to use _ to improve readability in constant name such as ULTIMATE_DESTINATION